test/api.test.js

```diff
@@ -1,5 +1,15 @@
-import test from 'ava'
+const newman = require('newman')
```

Comment:
we are writing in ES6 so please stay with that:

```suggestion
import newman from 'newman'
```